pkgs/development/python-modules/gpxpy/default.nix

```
@@ -2,14 +2,14 @@
 
 buildPythonPackage rec {
   pname = "gpxpy";
-  version = "1.4.2";
+  version = "1.5.0";
   disabled = !isPy3k;
```

```suggestion
  disabled = pythonOlder "3.6";
```

see https://github.com/tkrajina/gpxpy/blob/dev/setup.py#L40